tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -989,10 +991,10 @@ cc_library(
         "//tensorflow/core/platform:statusor",
         "//tensorflow/stream_executor:stream_header",
     ] + if_cuda_is_configured([
-        "//tensorflow/stream_executor/cuda:cublas_lt_header",
         "//tensorflow/core/platform/default/build_config:cublas_plugin",
         "//tensorflow/stream_executor:host_or_device_scalar",
         "//tensorflow/stream_executor:scratch_allocator",
+        "//tensorflow/stream_executor/cuda:cublas_lt_header",
```

Comment:
Is the re-arranging of order of deps required?

Reply:
It's not required and could be easily reverted.